src/bootstrap/lib.rs

```diff
@@ -854,12 +855,9 @@ impl Build {
     ///
     /// If no custom `llvm-config` was specified then Rust's llvm will be used.
     fn is_rust_llvm(&self, target: TargetSelection) -> bool {
-        if self.config.llvm_from_ci && target == self.config.build {
-            return true;
-        }
-
         match self.config.target_config.get(&target) {
-            Some(ref c) => c.llvm_config.is_none(),
+            Some(Target { llvm_is_vanilla: Some(is_vanilla), .. }) => !is_vanilla,
+            Some(Target { llvm_config, .. }) => llvm_config.is_none(),
```

Comment:
Seems like this is a behavior change -- previously llvm_from_ci would return true here, but I think this returns false now?

The way CI LLVM works *is* by setting up llvm-config.

Reply:
Oh I missed that it set llvm-config. Fixed.